pkgs/development/python-modules/shiboken6/default.nix

```diff
@@ -47,6 +54,10 @@ stdenv'.mkDerivation rec {
     "-DBUILD_TESTS=OFF"
   ];
 
+  postPatch = ''
+    substituteInPlace cmake/ShibokenHelpers.cmake --replace '#!/bin/bash' "#!$SHELL"
```

Comment:
Is this always set?

```
$ SHELL= bash -c 'echo $SHELL'

```

Reply:
I guess it must be otherwise it would break for you. In case it's empty, you could use buildPackages.bash if its a build time shell script or runtimeShell if it is a path that ends up in $out.

Reply:
It's a build-time shell that's needed here and stdenv [sets SHELL in the setup hook](https://github.com/NixOS/nixpkgs/blob/2ed7a5ab5962a9a88210900a2b3b24f8e8d9cba4/pkgs/stdenv/generic/builder.sh#L10) and stdenv actually [bails if it's not set](https://github.com/NixOS/nixpkgs/blob/2ed7a5ab5962a9a88210900a2b3b24f8e8d9cba4/pkgs/stdenv/generic/setup.sh#L439)

If you have a suggestion for something clearer or more idiomatic though I can change it!

Reply:
no should be fine than.